requirements-freeze.txt

```diff
@@ -52,7 +53,7 @@ pyutilib.component.core==4.5.3
 PyYAML==5.3.1
 redis==2.10.1
 repoze.lru==0.6
-repoze.who==1.0.18
+repoze.who==2.0
```

Comment:
@starsinmypockets just checking, in your ealier change to datagov-deploy, you had this at 2.2.0... which is correct?

Reply:
@adborden This is sort of weird - `requirements.txt` has `repoze.who==1.0.18` so I'm unclear who is bumps the major version, not what I would expect.